head_settling_duration: 4.0
bag_file_name: ar_target_frames.bag

confidence_threshold: 0.81

head_look_at_configurations:
# lower_pan, lower_tilt, target_x, target_y, target_z
  - look_at_configuration: [0.0, 1.1, -0.5, 0.95, 0.78]
  - look_at_configuration: [0.0, 1.1, 0.0, 0.9, 0.78]
  - look_at_configuration: [0.0, 1.1, 0.7, 0.8, 0.78]
  - look_at_configuration: [0.0, 1.1, 0.6, 1.1, 0.78]
  - look_at_configuration: [0.0, 1.1, 0.0, 1.1, 0.78]
  - look_at_configuration: [0.0, 1.1, -0.4, 1.1, 0.78]
  - look_at_configuration: [0.0, -1.0, -0.5, 0.95, 0.78]
  - look_at_configuration: [0.0, -1.0, 0.0, 0.9, 0.78]
  - look_at_configuration: [0.0, -1.0, 0.7, 0.8, 0.78]
  - look_at_configuration: [0.0, -1.0, 0.6, 1.1, 0.78]
  - look_at_configuration: [0.0, -1.0, 0.0, 1.1, 0.78]
  - look_at_configuration: [0.0, -1.0, -0.4, 1.1, 0.78]
  - look_at_configuration: [1.57, 1.0, -0.5, 0.95, 0.78]
  - look_at_configuration: [1.57, 1.0, 0.0, 0.9, 0.78]
  - look_at_configuration: [1.57, 1.0, 0.7, 0.8, 0.78]
  - look_at_configuration: [1.57, 1.0, 0.6, 1.1, 0.78]
  - look_at_configuration: [1.57, 1.0, 0.0, 1.1, 0.78]
  - look_at_configuration: [1.57, 1.0, -0.4, 1.1, 0.78]
  - look_at_configuration: [0.0, 0.0, -0.5, 0.95, 0.78]
  - look_at_configuration: [0.0, 0.0, 0.0, 0.9, 0.78]
  - look_at_configuration: [0.0, 0.0, 0.7, 0.8, 0.78]
  - look_at_configuration: [0.0, 0.0, 1.1, 0.6, 0.78]
  - look_at_configuration: [0.0, 0.0, 0.9, 1.0, 0.78]
  - look_at_configuration: [0.0, 0.0, 0.3, 1.0, 0.78]
  - look_at_configuration: [0.0, 0.0, -0.3, 1.0, 0.78]
  - look_at_configuration: [0.0, 0.0, -0.4, 1.1, 0.78]
  - look_at_configuration: [0.0, 0.0, 0.0, 1.1, 0.78]
  - look_at_configuration: [0.0, 0.0, 0.6, 1.1, 0.78]
  - look_at_configuration: [0.0, 0.0, 1.1, 1.1, 0.78]
  - look_at_configuration: [0.0, 0.0, 1.5, 1.1, 0.78]
  - look_at_configuration: [0.0, 0.0, 1.3, 1.5, 0.78]
  - look_at_configuration: [0.0, 0.0, 1.1, 1.7, 0.78]
  - look_at_configuration: [0.0, 0.0, 0.0, 1.7, 0.78]
  - look_at_configuration: [0.0, 0.0, -0.6, 1.7, 0.78]
  - look_at_configuration: [-1.57, 1.0, -0.5, 0.95, 0.78]
  - look_at_configuration: [-1.57, 1.0, 0.0, 0.9, 0.78]
  - look_at_configuration: [-1.57, 1.0, 0.7, 0.8, 0.78]
  - look_at_configuration: [-1.57, 1.0, 0.6, 1.1, 0.78]
  - look_at_configuration: [-1.57, 1.0, 0.0, 1.1, 0.78]
  - look_at_configuration: [-1.57, 1.0, -0.4, 1.1, 0.78]

#  - look_at_configuration: [-0.6, 0.9, 0.78]
#  - look_at_configuration: [-0.2, 0.9, 0.78]
#  - look_at_configuration: [0.25, 0.9, 0.78]
#  - look_at_configuration: [0.75, 0.9, 0.78]
#  - look_at_configuration: [0.75, 1.2, 0.78]
#  - look_at_configuration: [0.25, 1.2, 0.78]
#  - look_at_configuration: [-0.2, 1.2, 0.78]
#  - look_at_configuration: [-0.6, 1.2, 0.78]
#  - look_at_configuration: [-0.6, 1.45, 0.78]
#  - look_at_configuration: [-0.2, 1.45, 0.78]
#  - look_at_configuration: [0.3, 1.45, 0.78]
#  - look_at_configuration: [0.75, 1.45, 0.78]

lower_ptu_configurations:
  - ptu_configuration: [0.0, 1.1]
  - ptu_configuration: [0.0, -1.0]
  - ptu_configuration: [1.57, 1.0]
  - ptu_configuration: [0.0, 0.0]
  - ptu_configuration: [-1.57, 1.0]
